org.kde.krename.json

```diff
@@ -0,0 +1,67 @@
+{
+    "id": "org.kde.krename",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "krename",
+    "rename-icon": "krename",
+    "finish-args": [
+    	"--share=ipc", 
+    	"--socket=fallback-x11", 
+    	"--socket=wayland", 
+    	"--filesystem=host" 
+    ],
+
+    "modules": [
+        {
+            "name": "libexif",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2",
+                    "sha256": "d47564c433b733d83b6704c70477e0a4067811d184ec565258ac563d8223f6ae"
+                }
+            ]
+        },
+        {
+            "name": "taglib",
+            "config-opts": [ "-DBUILD_SHARED_LIBS=ON" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.github.io/releases/taglib-1.12.tar.gz",
+                    "sha256": "7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703"
+                }
+            ]
+        },
+        {
+            "name": "kjs",
+            "config-opts": [ "-DENABLE_TESTING=OFF" ],
```

Comment:
All array members should be on their own line. Same for above in taglib.

Reply:
yeah they are in there separate line already. If i am understanding correctly that you are asking for different elements in sources array.

Reply:
Yes